src/write_gcm.c

```diff
@@ -2455,10 +2576,12 @@ typedef struct {
 
 typedef struct {
   _Bool pretty_print_json;
+  _Bool enable_metadata_agent;
```

Comment:
The wg_configbuilder_t is the struct used to load configuration parameters when the plugin is loaded the first time. It isn't available after that.  At metrics dispatch time (in wg_transmit_unique_segments) I need both pieces of information so I need to propagate it through using the wg_context_t struct.